src/graph/serializers/serialize_schema.c

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2018-2019 Redis Labs Ltd. and Contributors
+ *
+ * This file is available under the Apache License, Version 2.0,
+ * modified with the Commons Clause restriction.
+ */
+
+#include "serialize_schema.h"
+#include "../../util/rmalloc.h"
```

Comment:
Delete; rmalloc unused in this file (which I think is the right call, given how triemaps manage memory).